drivers/clock_control/clock_stm32_ll_common.c

```diff
@@ -633,6 +633,23 @@ static void set_up_fixed_clock_sources(void)
 
 		z_stm32_hsem_unlock(CFG_HW_RCC_SEMID);
 	}
+
+#if defined(RCC_HSI48_SUPPORT)
```

Comment:
```suggestion
	if (IS_ENABLED(STM32_HSI48_ENABLED)) {
```

Reply:
Only stm32 devices  with STM32_HSI48_ENABLED  defined have the LL_RCC_HSI48_Enable in their LL drivers.
For example on stm32wl55, which has no RCC_HSI48_SUPPORT, has no LL_RCC_HSI48_Enable function